grafana/risingwave-dashboard.py

```diff
@@ -223,7 +223,11 @@ def section_compaction(outer_panels):
                     "avg(storage_compact_task_pending_num) by(job, instance)", "compactor_task_split_count - {{job}} @ {{instance}}"
                 ),
             ]),
-
+            panels.timeseries_count("Idle Compactor Count", [
```

Comment:
can we calculate the percentage of task_num occupy, like `compacting_task_num / global_max_max_concurrent_task_number` to  measure the occupy of task_assign

Reply:
Good Idea.